mcux/drivers/kinetis/CMakeLists.txt

```diff
@@ -36,6 +36,7 @@ zephyr_library_sources_ifdef(CONFIG_SPI_MCUX_LPSPI	fsl_lpspi.c)
 zephyr_library_sources_ifdef(CONFIG_UART_MCUX		fsl_uart.c)
 zephyr_library_sources_ifdef(CONFIG_UART_MCUX_LPSCI	fsl_lpsci.c)
 zephyr_library_sources_ifdef(CONFIG_UART_MCUX_LPUART	fsl_lpuart.c)
+zephyr_library_sources_ifdef(CONFIG_COUNTER_MCUX_PIT	fsl_pit.c)
```

Comment:
Please move this up to line 18 to preserve alpha order by CONFIG symbol.

Reply:
I see, the order is by the config, not by driver souce code. Thanks for this check. updated.